fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -53671,10 +53671,10 @@ is_a: FBbt:00007116 ! presumptive embryonic/larval system
 [Term]
 id: FBbt:00005710
 name: embryonic fat body
-def: "Fat body of the embryo." [FBC:SPR]
+def: "Fat body of the embryo from stage 16/17. This structure is a single-cell thick, lateral ribbon with dorsal fingers extending from the posterior region. Holes within the ribbon are formed by the intrusion of various organs (Hoshizaki et al., 1994)." [FBC:CP, FBC:SPR, FlyBase:FBrf0073371]
```

Comment:
Probably worth checking usage in curation. I suspect you'll find phenotype and expression curation with specified stages before S16.